### PR TITLE
Refix offset error: add back line that was deleted during a merge

### PR DESCRIPTION
**Pull request by reuvenlax**
This includes @ahmedabu98 's unit test to validate the fix.

R: @johnjcasey 
R: @ahmedabu98

### CODE REVIEW

**Comment by ahmedabu98** [comment]
Run Java_GCP_IO_Direct PreCommit

**Comment by ahmedabu98** [comment]
FYI @Abacn noticed the test might be flaky ([comment](https://github.com/apache/beam/pull/26559#issuecomment-1563445450))

**Comment by Abacn** [comment]
Thanks @ahmedabu98 Let's see if that unit test is fixed with this PR

File: sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOWriteTest.java [hunk 1]
```diff
@@ -3025,4 +3025,50 @@ public void testWriteWithStorageApiWithoutSettingShardsEnableAutoSharding() thro
         fakeDatasetService.getAllRows("project-id", "dataset-id", "table-id"),
         containsInAnyOrder(new TableRow().set("name", "a"), new TableRow().set("name", "b")));
   }
+
+  @Test
+  public void testBatchStorageWriteWithMultipleAppendsPerStream() throws Exception {
+    assumeTrue(useStorageApi);
+    assumeTrue(!useStreaming);
```
Comment (Abacn) [inline sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOWriteTest.java:3032]: as in that pr, to make this test run one needs to add a parametrized combination satisfies these constraints

Comment (Abacn) [reply sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOWriteTest.java:3032]: never mind

I see the test is executed: https://ci-beam.apache.org/job/beam_PreCommit_Java_GCP_IO_Direct_Phrase/181/testReport/org.apache.beam.sdk.io.gcp.bigquery/BigQueryIOWriteTest/testBatchStorageWriteWithMultipleAppendsPerStream_2_/

Comment (Abacn) [reply sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOWriteTest.java:3032]: Current succeeded run:
https://ci-beam.apache.org/job/beam_PreCommit_Java_GCP_IO_Direct_Commit/2745/
https://ci-beam.apache.org/job/beam_PreCommit_Java_GCP_IO_Direct_Phrase/181/

**Comment by Abacn** [comment]
Run Java_GCP_IO_Direct PreCommit

**Pull request merged**